src/main/app/drafts/draft-data/claimDraft.ts

```diff
@@ -11,37 +11,37 @@ export function prepareClaimDraft (userEmailAddress: string) {
     claimant: {
       partyDetails: {
         type: PartyType.INDIVIDUAL.value,
-        name: 'Jack Davids',
+        name: 'Jan Clark',
         address: {
-          line1: 'Flat 1',
-          line2: 'Street 1',
-          line3: 'Cool house name',
+          line1: 'Street 1',
+          line2: 'Street 2',
+          line3: 'Street 3',
           city: 'London',
-          postcode: 'SW1A 2AA'
+          postcode: 'SW1A 9AA'
         },
         dateOfBirth: {
           known: true,
           date: {
-            year: 1997,
-            month: 5,
-            day: 27
+            year: 2000,
+            month: 1,
+            day: 1
           }
         }
       },
       mobilePhone: {
-        number: '07112358132'
+        number: '(0)207 127 0000'
```

Comment:
isn't this not a valid mobile?

Reply:
I asked this to pragnesh at the time and he said it was fine

Reply:
has it been tested? I don't think it is fine

Reply:
@timja @jacmurphy - Its asking a claimant for a telephone number - The field is free text and was changed to that, so I think its valid to add a telephone number -  I can refactor it, but I just did the minimum change here

Reply:
actually you're right 👍